### PR TITLE
Create test trunk

### PR DESCRIPTION
**Pull request by Pavlova78**

### CODE REVIEW

File: test trunk [hunk 1]
```diff
@@ -0,0 +1 @@
+Hi trunk!
```
Comment (OkhremukAlexei) [inline test trunk]: Pls remove this line

Comment (OkhremukAlexei) [inline test trunk]: Check

**Review by OkhremukAlexei — CHANGES_REQUESTED** [review]

File: test trunk (hunk 1 above)
Comment (Pavlova78) [reply test trunk:1]:
```suggestion
Hi trunk!
```

Comment (Pavlova78) [reply test trunk]:
```suggestion
Hi trunk! @@@@
```

**Review by OkhremukAlexei — APPROVED** [review]

**Pull request merged**